go/protocol/keybase1/extras.go

```diff
@@ -1454,6 +1454,14 @@ func (u UserVersion) Eq(v UserVersion) bool {
 	return u.Uid.Equal(v.Uid) && u.EldestSeqno.Eq(v.EldestSeqno)
 }
 
+type ByUserVersionID []UserVersion
```

Comment:
I thought this might be useful in more than one place, in which case I think it is easier to just have the traditional sort mechanic be available.